components/DBTableMocker.php

```diff
@@ -10,11 +10,18 @@
     class DBTableMocker {
 
         /**
-         * Name of the column which is being changed
+         * Contains an array of stored table columns
          *
-         * @var string
+         * @var string[][]
          */
-        private $currentColumn = "";
+        private $columns = [];
+
+        /**
+         * Column which is being changed
+         *
+         * @var string[]
+         */
+        private $currentColumn = [];
```

Comment:
Current column is now an array which contains name and config of the column which is being configured.